network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -229,81 +228,41 @@ <h3 class="h3-heading">{% trans "Can I control it?" %}</h3>
-                {# TODO: Add the privacy ding here #}
-                <img src="" alt="{% trans 'This product does not meet our minimum security standards' %}" />
-              {% endif %}
-
-              {% trans "Overall Security Rating" as overall_security_rating %}
-              {% include "fragments/product_criterion_primary_info.html"  with label=overall_security_rating %}
-              <div>
-                <span class="security-score mb-0">{{security_score|localize}}</span><span class="full-security-score mb-0">/{{full_security_score}}</span>
-                <img
-                  src="{% static "_images/buyers-guide/score/icon-star.svg" %}"
-                  width="20"
-                  height="20"
-                  alt="{% trans "star" context "Used as a noun. Accessibility label for star image" %}"
-                >
-              </div>
-            </div>
-            {% trans "Encryption"              as encryption %}
-            {% trans "Security updates"        as security_updates %}
-            {% trans "Strong password"         as strong_password %}
-            {% trans "Manages vulnerabilities" as manages_vulnerabilities %}
-            {% trans "Privacy policy"          as privacy_policy %}
-
-            <h2 class="h3-heading">Does this product meet our <u>Minimum Security Standards?</u></h2>
-            {% if product.meets_minimum_security_standards %}
-              {% trans "Yes" %}
-            {% else %}
-              {% trans "No" %}
-            {% endif %}
-
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.uses_encryption         help=product.uses_encryption_helptext         label=encryption                %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.strong_password         help=product.strong_password_helptext         label=strong_password           %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.security_updates        help=product.security_updates_helptext        label=security_updates          %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.manage_vulnerabilities  help=product.manage_vulnerabilities_helptext  label=manages_vulnerabilities   %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.privacy_policy          help=product.privacy_policy_helptext          label=privacy_policy            %}
-          </div>
-        </div>
-      </div>
+      <section class="minimum-security criterion-group">
+        <h2 class="h3-heading">{% trans "Security" %}</h2>
+        {% trans "Does this product meet our <a href='https://foundation.mozilla.org/privacynotincluded/about/meets-minimum-security-standards'>Minimum Security Standards</a>?" as minimum_security_standards %}
```

Comment:
This one is... tricky. @TheoChevalier do you know if we can blocktrans here, to keep that `url` from being part of the translation string itself?

Reply:
Tricky indeed, but turns out it’s possible using `asvar`!
Filed a PR to be merged with this one here https://github.com/mozilla/foundation.mozilla.org/pull/5347

Reply:
thanks!